docs/MESH_PROTOCOL.md

```diff
@@ -156,7 +166,7 @@ uint8_t | List size | 1 | Size of tracked devices list.
 
 Type | Name | Length | Description
 --- | --- | --- | ---
-uint8_t | Stone ID | 1 | ID of stone that requests for a sync.
+stone_id_t | Stone ID | 1 | ID of stone that requests for a sync.
```

Comment:
I'd prefer not to have typedefs in the protocol doc.

Reply:
Ah, I just made it consistent with the rest. :-)

Reply:
Okay, `s/stone_id_t/uint8_t/g` and at the same time added `_t` postfix where this was forgotten.

Reply:
Ah, right, the mesh protocol doc isn't on par with the protocol doc: it's copy pasted structs basically.

Reply:
We don't use the _t prefix in the protocol doc though, but well, i'm fine with this.